src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```
@@ -84,4 +95,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     </When>
   </Choose>
 
+  <Target Name="_CopyDefaultGlobalUsingFiles" AfterTargets="Restore" Condition="'$(EnableGlobalUsingFiles)' == 'true'">
+    <Copy
+      SourceFiles="%(DefaultGlobalUsingFiles.FullPath)"
+      DestinationFolder="$(GlobalUsingFilesDirectory)"
+      Condition="Exists('%(DefaultGlobalUsingFiles.FullPath)') And !Exists('$(GlobalUsingFilesDirectory)%(DefaultGlobalUsingFiles.Filename)%(DefaultGlobalUsingFiles.Extension)')" />
+  </Target>
+
+  <Target Name="_AddDefaultGlobalUsingFilesToCompile" BeforeTargets="BeforeCompile" DependsOnTargets="_CopyDefaultGlobalUsingFiles" Condition="'$(EnableGlobalUsingFiles)' == 'true'">
+    <ItemGroup>
+      <Compile Include="$(GlobalUsingFilesDirectory)*.cs" />
```

Btw, if we're dropping the file copying stuff, you can Compile Include it from the location in the SDK. 
```suggestion
      <Compile Include="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.Web.DefaultGlobalUsing.cs" />
```